src/api/Feed.js

```diff
@@ -0,0 +1,1022 @@
+     * @param {Task} task - The task for which the assignment create failed
+     * @param {Function} errorCallback - Passed in error callback
+     * @return {void}
+     */
+    createTaskAssignmentErrorCallback(e: $AxiosXHR<any>, file: BoxItem, task: Task, errorCallback: Function) {
+        this.errorCallback(e);
+        errorCallback(e);
+        // Attempt to delete the task due to it's bad assignment
+        this.deleteTask(file, task.id);
+    }
+
+    /**
+     * Deletes a task
+     *
+     * @param {BoxItem} file - The file to which the task is assigned
+     * @param {string} taskId - The task's id
+     * @param {Function} successCallback - the function which will be called on success
+     * @param {Function} errorCallback - the function which will be called on error
+     * @return {void}
+     */
+    deleteTask = (
+        file: BoxItem,
+        taskId: string,
+        successCallback: (taskId: string) => void = noop,
+        errorCallback: (e: $AxiosXHR<any>, taskId: string) => void = noop
+    ) => {
+        if (!file.id) {
+            throw getBadItemError();
+        }
+
+        this.id = file.id;
+        this.tasksAPI = new TasksAPI(this.options);
+        this.updateFeedItem({ isPending: true }, taskId);
+
+        this.tasksAPI.deleteTask({
+            file,
+            taskId,
+            successCallback: () => {
+                this.deleteFeedItem(taskId, successCallback);
+            },
+            errorCallback: (e: $AxiosXHR<any>) => {
+                this.errorCallback(e);
+                if (!this.isDestroyed()) {
+                    errorCallback(e, taskId);
+                }
+            }
+        });
+    };
+
+    /**
+     * Deletes a feed item from the cache
+     *
+     * @param {string} id - The id of the feed item to be deleted
+     * @param {Function} successCallback - function to be called after the delete
+     */
+    deleteFeedItem = (id: string, successCallback: Function = noop) => {
+        const cachedItems = this.getCachedItems(this.id);
+        if (cachedItems) {
+            const feedItems = cachedItems.items.filter((feedItem) => feedItem.id !== id);
+            this.setCachedItems(this.id, feedItems);
+
+            if (!this.isDestroyed()) {
+                successCallback(id);
+            }
+        }
+    };
+
+    /**
+     * Network error callback
+     *
+     * @param {Error} error - Axios error object
+     * @return {void}
+     */
+    errorCallback = (error: $AxiosXHR<any>): void => {
+        console.error(error); // eslint-disable-line no-console
+        this.hasError = true;
+    };
+
+    /**
+     * Fetches the task assignments for each task
+     *
+     * @param {Array} tasksWithoutAssignments - Box tasks
+     * @return {Promise}
+     */
+    fetchTaskAssignments(tasksWithoutAssignments: Tasks): Promise<?Tasks> {
+        const requestData = {
+            params: {
+                fields: TASK_ASSIGNMENTS_FIELDS_TO_FETCH.toString()
```

Comment:
Could you assign the stringed version of `TASK_ASSIGNMENTS_FIELDS_TO_FETCH ` to a const, and reuse it?

Reply:
This was copy + paste from another file, but saw the only thing which referenced this was in this file, so just ```.toString```'d it in the fields file and exported that 👍